src/fetch_real_estate_prices.py

```diff
@@ -0,0 +1,126 @@
+import json
```

Comment:
[`resquests`](http://docs.python-requests.org/en/master/) isn't native but I guess it's installed by default in Anaconda Python. The native module is [`urllib.request`](https://docs.python.org/3.5/library/urllib.request.html#module-urllib.request) — easy to mess up, isn't it? ; )